.github/workflows/pkgci_test_amd_mi250.yml

```diff
@@ -49,6 +49,13 @@ jobs:
           ./build_tools/pkgci/setup_venv.py ${VENV_DIR} \
             --artifact-path=${PACKAGE_DOWNLOAD_DIR} \
             --fetch-gh-workflow=${{ inputs.artifact_run_id }}
+      - name: "Installing build dependencies"
+        run: |
+          sudo apt update
+          sudo apt install cmake clang ninja-build
+          export CC=clang
+          export CXX=clang++
```

Comment:
Ohhhh wait, I should be using `>> $GITHUB_ENV`. Need to update a bunch of files o_o

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#environment-files